pageserver/src/tenant_mgr.rs

```diff
@@ -247,3 +248,22 @@ fn list_tenantids() -> Result<Vec<ZTenantId>> {
         })
         .collect()
 }
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct TenantInfo {
+    pub id: String,
+    pub state: String,
+}
+
+pub fn list_tenants() -> Result<Vec<TenantInfo>> {
+    let m = access_tenants();
+    m.iter()
+        .map(|v| {
+            let (id, tenant) = v;
+            Ok(TenantInfo {
+                id: id.to_string(),
```

Comment:
Not strongly necessary, but might look cleaner: `ZTenantId` derives `Serialize`, making the `TenantState` serializable is also trivial, so you can avoid string creations entirely here, letting `serde` to do its job instead.

But up to you.